### DISCUSSION TITLE
failed to cache strings

### DISCUSSION THREAD

**Discussion by nexovec** [a-h/templ#869] [Q&A]
Hi,
anybody has an idea why I could be getting this rendered instead of a templte?

`templ: failed to cache strings: templ: failed to stat /app/internal/templates/v1/common_templ.txt: stat /app/internal/templates/v1/common_templ.txt: no such file or directory`

I don't really know how to reproduce this or what it even means. I don't use any .txt file anywhere and it's been working just fine for the past 10 months. What is this?

**Comment by nexovec** [comment]
It seems that this sometimes happens when I use `templ generate` and `templ generate --watch` interchangeably. Not sure if this is a bug or not.

**Reply by ltdangle** [reply]
I'm having the same issue as well. I would get this error on remote server when the project is built with `templ generate --watch`.
When built with `templ generate` it works as expected.

**Comment by augustoguerrero** [comment]
Hi, I'm also having the same problem. I deleted a directory for a view that i no longer needed when running my project. Later when i run it again, i got this error:

templ: failed to cache strings: templ: failed to open /views/components/products_templ.txt

I did use templ generate and the --watch version interchangeably too. But i can't find the way to get rid of it.

**Comment by patrickkdev** [comment]
I'm also having this problem and don't know how to fix
